.github/workflows/tests.yml

```diff
@@ -97,7 +97,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        sdk-version: ["2.8.2-0-gfc96d10f5-r421"]
+        sdk-version: ["2.8.2-0-gfc96d10f5-r421", "2.10.2-0-gf4228cb7d-r508-linux-x86_64"]
```

Comment:
I propose to use only one version of the sdk for the tests, the latest one.

Reply:
I left only `["2.10.2-0-gf4228cb7d-r508-linux-x86_64"]`